code/game/objects/structures/janicart.dm

```diff
@@ -325,4 +329,45 @@
 /obj/structure/bed/chair/janicart/secway/update_mob()
 	if(buckled_mob)
 		buckled_mob.dir = dir
-		buckled_mob.pixel_y = 4
\ No newline at end of file
+		buckled_mob.pixel_y = 4
+
+//atv//
+
+/obj/structure/bed/chair/janicart/atv

```

Comment:
yeah it's especially weird considering the newer janitorial_cart is 
/obj/structure/janitorialcart
there is no logical reason for jancart to be that way